message-me/app/javascript/packs/application.js

```diff
@@ -16,15 +16,27 @@ ActiveStorage.start()
 window.scroll_bottom = function(id) {
   const element = $(`#${id}`)
   if (element.length > 0){
-  const scrollHeight = element[0].scrollHeight;
+    console.log("Hi")
+    const scrollHeight = element[0].scrollHeight;
     element.scrollTop(scrollHeight);
   }
 };
 
+function clear_input() {
+  const input = $('#messages_body')
```

Comment:
where is `messages_body` defined?

Reply:
in app/views/chatrooms/index.html.erb
![image](https://user-images.githubusercontent.com/67599130/126994108-9657111a-a43e-48e5-a806-df420c7ce462.png)

in browser
![image](https://user-images.githubusercontent.com/67599130/126993967-602f1029-6f81-4d14-874f-14d726586c16.png)

Reply:
Hate your implementation here.

https://trello.com/c/ZNYDswUt
You will understand why once this card is passed

Reply:
I just noticed one thing. Please explicitly state the ID in the `.html.erb` file.
Tell me why I ask you to say so

Reply:
so that other devs that see the code can understand where it is immediately rather than having to inspect with dev tools?

Reply:
so that other devs that see the code can understand where it is immediately rather than having to inspect with dev tools

Reply:
yes. that is right.
Code first for humans.